plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -65,7 +65,9 @@
         type: str
         required: True
       rf_profile:
-        description: Radio Frequency (RF) profile of the Access Point. For example, "HIGH".
+        description: |
+          Radio Frequency (RF) profile name of the Access Point either "HIGH" or "LOW" or "TYPICAL"
```

Comment:
Specifies the Radio Frequency (RF) profile name for the Access Point. It can be one of the standard profiles 
    "HIGH", "LOW", "TYPICAL", or a custom profile that has been created. For example, "HIGH".